src/components/faq.js

```
@@ -0,0 +1,83 @@
+        "@type": "FAQPage",
+        mainEntity: allQuestionAnswer,
+      });
+    }
+  }, [faq]);
+  return (
+    <>
+      {faqSchema && (
+        <Head>
+          <script
+            type="application/ld+json"
+            dangerouslySetInnerHTML={{ __html: JSON.stringify(faqSchema) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)